diff --git a/config.def.h b/config.def.h
index 0e01717..c0cbb24 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "xos4 Terminus:pixelsize=14:antialias=false:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -24,7 +24,7 @@ char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 char *vtiden = "\033[?6c";
 
 /* Kerning / character bounding-box multipliers */
-static float cwscale = 1.0;
+static float cwscale = 0.9;
 static float chscale = 1.0;
 
 /*
@@ -84,25 +84,30 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
 
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	/* Gruvbox Dark colorscheme: bright line of the palette */
+
+	/* 8 normal colors */
+	/* Used for font coloring */
+	"#1d2021", /* black */
+	"#fb4934", /* red */
+	"#b8bb26", /* green */
+	"#fabd2f", /* yellow */
+	"#83a598", /* blue */
+	"#d3869b", /* magenta */
+	"#8ec07c", /* cyan */
+	"#fbf1c7", /* white */
+
+	/* 8 bright colors (actually the same as normals, except black) */
+	/* Used for character background coloring */
+	"#504945", /* black */
+	"#fb4934", /* red */
+	"#b8bb26", /* green */
+	"#fabd2f", /* yellow */
+	"#83a598", /* blue */
+	"#d3869b", /* magenta */
+	"#8ec07c", /* cyan */
+	"#fbf1c7", /* white */
 
 	[255] = 0,
 
@@ -118,8 +123,8 @@ static const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
@@ -134,8 +139,8 @@ static unsigned int cursorshape = 2;
  * Default columns and rows numbers
  */
 
-static unsigned int cols = 80;
-static unsigned int rows = 24;
+static unsigned int cols = 118;
+static unsigned int rows = 25;
 
 /*
  * Default colour and shape of the mouse cursor
@@ -156,8 +161,8 @@ static unsigned int defaultattr = 11;
  */
 static MouseShortcut mshortcuts[] = {
 	/* button               mask            string */
-	{ Button4,              XK_ANY_MOD,     "\031" },
-	{ Button5,              XK_ANY_MOD,     "\005" },
+	{ Button4,              XK_NO_MOD,      "\031" },
+	{ Button5,              XK_NO_MOD,      "\005" },
 };
 
 /* Internal keyboard shortcuts. */
@@ -173,6 +178,9 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_Prior,       zoom,           {.f = +1} },
 	{ TERMMOD,              XK_Next,        zoom,           {.f = -1} },
 	{ TERMMOD,              XK_Home,        zoomreset,      {.f =  0} },
+	{ TERMMOD,              XK_KP_Prior,    zoom,           {.f = +1} },
+	{ TERMMOD,              XK_KP_Next,     zoom,           {.f = -1} },
+	{ TERMMOD,              XK_KP_Home,     zoomreset,      {.f =  0} },
 	{ TERMMOD,              XK_C,           clipcopy,       {.i =  0} },
 	{ TERMMOD,              XK_V,           clippaste,      {.i =  0} },
 	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
